spec/features/admin/enrollments/show_spec.rb

```diff
@@ -25,5 +27,17 @@
         expect(page).to have_content(student.phone)
       end
     end
+
+    it 'shows a trash button, an pen button, and a save button(disabled by default) for every student' do
+      expect(page).to have_button("trash", count: 20)
+      expect(page).to have_button("pen", count: 20)
+      expect(page).to have_button("save", disabled: true, count: 20)
+    end
+
+    it 'shows a a print button in the enrollment details ' do
```

Comment:
I'll see further if we can mock/stub a print request, I would be surprised if there is not a solution out there.